java/ql/src/Performance/NewStringString.ql

```diff
@@ -6,8 +6,10 @@
  * @problem.severity recommendation
  * @precision high
  * @id java/inefficient-string-constructor
- * @tags efficiency
- *       maintainability
+ * @tags quality
+ *       reliability
+ *       performance
```

Comment:
Putting this in `performance` is perhaps a bit of a stretch. It's probably more like a bad practice.